src/Components/Navbar.jsx

```diff
@@ -35,7 +35,7 @@ export default function Navbar() {
         <>
             <div className="navbar sticky-top navbar-expand-lg navbar-light navcus frosted-glass" style={navbarStyle}>
                 <div className="container-fluid" >
-                    <a className="navbar-brand" style={navBrandStyle}  >
+                    <a href="https://gdscsmit.github.io/" className="navbar-brand" style={navBrandStyle}  >
```

Comment:
Please use Home Route i.e. `/` in place of using `anchor tag`
Refer [here](https://v5.reactrouter.com/web/api/Link)
@avg77